ppr-ui/src/router/routes.ts

```diff
@@ -148,6 +149,56 @@ export const routes = [
       requiresAuth: true
     }
   },
+  {
+    path: '/mhr-registration/your-home',
+    name: RouteNames.YOUR_HOME,
+    component: MhrRegistration,
+    meta: {
+      step: 1,
+      label: 'Describe your Home',
+      requiresAuth: true
+    }
+  },
```

Comment:
Slightly different approach than PPR.
Chose to use a single base component  `MhrRegistration ` and template the steps from that single component. 
This was to prevent code duplication (like the fee summary, stepper and bottom bar action buttons) across the various steps and can just exist in one place.